altair/vega/v2/display.py

```diff
@@ -1,6 +1,10 @@
 import os
 
-from IPython.display import display
+try:
+    from IPython.display import display
+    IPYTHON_AVAILABLE=True
+except ImportError:
+    IPYTHON_AVAILABLE=False
```

Comment:
Here and in the other three similar places, I'd prefer to do the import within the ``vega()`` function rather than with a try/except at the top.

As currently written, if a user does not have IPython installed and tries to call ``vega()``, the result will be a ``NameError``, and they'll have read through the source code and reason that ``import IPython`` must have given an ``ImportError`` which led to ``display`` being undefined.

If we just import ``IPython`` where it's used, the result will be much clearer for the user, and it also makes the source code easier to follow.

Reply:
✅ done